html/semantics/forms/the-select-element/common-HTMLOptionsCollection-namedItem.html

```diff
@@ -37,26 +39,16 @@
 }, "if no item has a name or id value matching the parameter, return null and stop");
 
 test(function () {
-    var testarr = [];
-    for (var i = 0; i < selly.namedItem('id3').length; i++) {
-        testarr.push(selly.namedItem('id3')[i].text);
-    }
-    assert_array_equals(testarr, ['3','duplicate ID']);
-}, "return an HTMLOptionsCollection in correct order for repeated 'id' value");
+    assert_equals(selly.namedItem('id3')["value"], "3");
+}, "if multiple items have a name or *id* value matching the parameter, return the first object and stop");
 
 test(function () {
-    var testarr = [];
-    for (var i = 0; i < selly.namedItem('name4').length; i++) {
-        testarr.push(selly.namedItem('name4')[i].text);
-    }
-    assert_array_equals(testarr, ['4', 'duplicate name']);
-}, "return an HTMLOptionsCollection in correct order for repeated 'name' value");
+    assert_equals(selly.namedItem('name4')["value"], "4");
+}, "if multiple items have a *name* or id value matching the parameter, return the first object and stop");
 
 test(function () {
-    var testarr = [];
-    for (var i = 0; i < selly.namedItem('mixed1').length; i++) {
-        testarr.push(selly.namedItem('mixed1')[i].text);
-    }
-    assert_array_equals(testarr, ['mixed ID', 'mixed name']);
-}, "return an HTMLOptionsCollection in correct order for repeated mixed value");
+    assert_equals(selly.namedItem('mixed1')["value"], "mixed ID");
```

Comment:
The parameter quoted by Ryosuke is about the behavior of the named property getter on HTMLOptionsCollection [1]. Since HTMLSelectElement.namedItem() calls the named property getter on the options collection [2], I believe it applies (I agree with Ryosuke).

Regarding https://github.com/w3c/web-platform-tests/blob/master/html/semantics/forms/the-select-element/select-named-getter.html, I believe the test is correct. HTMLSelectElement does NOT have a named property getter (notice the absence of 'getter' before the namedItem() operation). Therefore, you cannot access the options collection named properties as if they were regular properties on the HTMLSelectElement.

[1] https://html.spec.whatwg.org/multipage/infrastructure.html#htmloptionscollection
[2] https://html.spec.whatwg.org/multipage/forms.html#dom-select-nameditem